test-one/src/index.js

```diff
@@ -1,14 +1,24 @@
 import { log } from "./helpers/logger";
 
 export const go = async () => {
-  // Your code should go here instead of the welcome message below. Create and modify additional files as necessary/appropriate.
+    var counter;
+    var displayMessage = "";
 
-  const welcomeMessage =
-    "Congrats, you're up and running! Welcome to the first moshtix code challenge! " +
-    "Please replace this message with your solution and feel free to add " +
-    "or modify other files as necessary. Good luck : )";
+    for (counter = 1 ; counter <= 100; counter++) {
+        if (counter % 3 && counter % 5) {
```

Comment:
Can you think of another way of writing this line of code?

Reply:
Yes, I could of used some switch statement or while loops like below:

import { log } from "./helpers/logger";

export const go = async () => {
    var counter = 1;
    var displayMessage = "";

    while (counter <= 100) {
        switch (counter % 3 && counter % 5) {
            default: displayMessage += counter; break;
            case 0:
                if (counter % 3 == 0) {
                    displayMessage += "Boss";
                }
                if (counter % 5 == 0) {
                    displayMessage += "Hog";
                }; break;     
        }
        displayMessage += "\n";
        counter++
    }

    await log(displayMessage);
};

go();